proto/redpanda/api/dataplane/v1alpha1/secret.proto

```diff
@@ -24,7 +24,12 @@ message Secret {
 }
 
 message SecretInput {
-  string id = 1 [(google.api.field_behavior) = IMMUTABLE];
+  string id = 1 [
+    (google.api.field_behavior) = IMMUTABLE,
```

Comment:
Could possibly remove this since this type is exclusively used for creating secrets and not reused for retrieving or updating right?